apps/site/src/pages/settings/billing-settings-panel/billing-overview.tsx

```diff
@@ -41,7 +41,7 @@ export const BillingOverviewPanelPage: FunctionComponent = () => {
       user.stripeSubscriptionStatus &&
       !user.stripeSubscriptionId
     ) {
-      setTimeout(() => refetch(), 1000);
+      setTimeout(() => refetch(), 5000);
```

Comment:
The comment above should reflect the new time (5 seconds)

Reply:
Addressed in https://github.com/blockprotocol/blockprotocol/pull/1068/commits/4f2963453016bc61effb0b078ba3eeea603c4854